app/feedperson/utils.py

```diff
@@ -19,6 +19,13 @@ def load_feed_people():
   for person in people:
     location = person.location.fordisp.text
     if (re.match('114 Western Ave', location) or re.match('150 Western Ave', location)):
+      
+      # This regex is a temp fix whilst the location data is inconsistent in PeopleSoft.
+      # Should be removed once the data is fixed.
+      if re.match('150 Western Ave', location):
+        location = re.sub(r'Sci&Eng', 'SEC', location)
+        location = re.sub(r'150 Western Avenue', '150 Western Ave, SEC', location)
```

Comment:
This is a minor point (and, again, something that really be fixed at the data source) but all of the "114 Western Ave" entries have a period after "Ave" (e.g. "114 Western Ave. 3112.8"). Is that worth adding the period to these? 

Also, there's some inconsistency in the "SEC" entries, where some of them give the room as "SEC 3.123", while other have it as "SEC **Room** 3.123". Is it worth regexing out the "Room" as well?

Reply:
I think we should see how this looks once we get it all set up. If it ends up looking very odd, we can change at a later date. Generally though, I agree, this should be changed at the source.